.github/workflows/ci.yml

```diff
@@ -115,6 +115,69 @@ jobs:
       - name: Integration tests
         run: pytest -n12 -vv tests/integration
 
+  test_against_latest_sentry:
+    name: Sentry-Symbolicator Integration Tests
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Symbolicator
+        uses: actions/checkout@v2
+
+      - name: Install rust stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+          override: true
+
+      - name: Cache rust cargo artifacts
+        uses: swatinem/rust-cache@v1
+
+      - name: Build Relay
+        run: cargo build --locked
+
+      - name: Run Relay
+        run: ./target/debug/symbolicator run &
+
+      # Checkout Sentry and run integration tests against latest Symbolicator
+      - name: Checkout sentry
+        uses: actions/checkout@v2
+        with:
+          repository: getsentry/sentry
+          path: sentry
+
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.6
+
+      - name: Setup steps
+        id: setup
+        run: |
+          pip install --upgrade pip wheel
+          echo "::set-output name=pip-cache-dir::$(pip cache dir)"
+          # We cannot execute actions that are not placed under .github of the main repo
+          mkdir -p .github/actions/setup-sentry/
+          cp sentry/.github/actions/setup-sentry/action.yml .github/actions/setup-sentry/action.yml
+
+      - name: Sentry's pip cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.setup.outputs.pip-cache-dir }}
+          key: sentry-deps-${{ hashFiles('sentry/requirements**.txt') }}
+          restore-keys: sentry-deps-
+
+      - name: Setup Sentry
+        uses: ./.github/actions/setup-sentry
+        with:
+          workdir: sentry
+          snuba: true
+          kafka: true
```

Comment:
Does symbolicator need these?

Reply:
Let us try. I grabbed it from Sentry's workflow.

Reply:
symbolicator itself does not use kafka at all.  it only receives http requests from the `symbolicate_event` celery task.

Reply:
It definitely did not like the change and run for 6 hours:
<img width="1048" alt="image" src="https://user-images.githubusercontent.com/44410/126492813-3f509ed7-a210-4bca-86b1-b8271c7b628c.png">

Reply:
We need both services:
<img width="865" alt="image" src="https://user-images.githubusercontent.com/44410/126550924-38ab86f8-901b-4460-aae0-0b66b05da64c.png">